x-pack/plugins/ml/server/routes/modules.ts

```diff
@@ -26,42 +30,16 @@ export function dataRecognizer(
   { router, routeGuard }: RouteInitialization,
   compatibleModuleType: CompatibleModule | null
 ) {
-  /**
-   * @apiGroup Modules
-   *
-   * @api {get} /internal/ml/modules/recognize/:indexPatternTitle Recognize index pattern
-   * @apiName RecognizeIndex
-   * @apiDescription By supplying an index pattern, discover if any of the modules are a match for data in that index.
-   * @apiSchema (params) recognizeModulesSchema
-   * @apiSchema (query) moduleFilterSchema
-   * @apiSuccess {object[]} modules Array of objects describing the modules which match the index pattern, sorted by module ID.
-   * @apiSuccessExample {json} Success-Response:
```

Comment:
What sort of info will be displayed now for an example response like this? Just wondering as these examples from the modules route are quite rich in information in showing you the structure of a module.